include/quicr/detail/messages.h

```diff
@@ -172,6 +174,19 @@ namespace quicr::messages {
     BytesSpan operator>>(BytesSpan buffer, MoqServerSetup& msg);
     Bytes& operator<<(Bytes& buffer, const MoqServerSetup& msg);
 
+    //
+    // New Group
+    //
+
+    struct NewGroupRequest
+    {
+        uint64_t subscribe_id;
+        uint64_t track_alias;
```

Comment:
we don't need track alias here

Reply:
The reason is that the client must have the track alias to match received objects. The client maintains the track alias and can provide it. The server has the track alias as well, but forwarding of new group goes the the origin publisher which may not even be on the same relay. Forwarding to the origin publisher involves looking up the track alias. **The track alias is required**.  

That being said, we can of course take a performance hit and do a double lookup for every new group request. The double lookup is to lookup the subscribe ID of the incoming request to find the track alias that the subscribe ID is associated to. Then to do what we would have done had the original request had track alias.